www/index.html

```diff
@@ -29,9 +29,9 @@
 
         <div class="row">
             <div class="about-chabble col-xs-12 collapse" id="about-chabble">
-                <h4>Welcome to chatjs</h4>
+                <h4>Welcome to Chabble</h4>
```

Comment:
@Harxy Ok - mega pedantic here, but is the app chabble or Chablle.  To date we have used lower case so maybe should agree a standard.